Description: Name substitutions
 Substitute 'freeradius' for 'radiusd' as the daemon name and 'freerad' for 
 'radius' as the user and group name
Author: Alan DeKok <aland@freeradius.org>,
 Adrien Demarez,
 Arran Cudbard-Bell <a.cudbardb@freeradius.org>,
 Matthew Newton, Fajar Arief Nugraha <github@fajar.net>
Origin: https://github.com/FreeRADIUS/freeradius-server/blob/v3.1.x/debian/patches/radiusd-to-freeradius.diff
Last-Update: 2014-05-19

--- a/Make.inc.in
+++ b/Make.inc.in
@@ -95,7 +95,7 @@
 
 LOGDIR		= ${logdir}
 RADDBDIR	= ${raddbdir}
-RUNDIR		= ${localstatedir}/run/radiusd
+RUNDIR		= ${localstatedir}/run/freeradius
 SBINDIR		= ${sbindir}
 RADIR		= ${radacctdir}
 LIBRADIUS	= $(top_builddir)/src/lib/$(LIBPREFIX)freeradius-radius.la $(TALLOC_LIBS)
--- a/raddb/radiusd.conf.in
+++ b/raddb/radiusd.conf.in
@@ -61,7 +61,7 @@
 
 #
 #  name of the running server.  See also the "-n" command-line option.
-name = radiusd
+name = freeradius
 
 #  Location of config and logfiles.
 confdir = ${raddbdir}
@@ -416,8 +416,8 @@
 	#  member.  This can allow for some finer-grained access
 	#  controls.
 	#
-#	user = radius
-#	group = radius
+	user = freerad
+	group = freerad
 
 	#  Core dumps are a bad thing.  This should only be set to
 	#  'yes' if you're debugging a problem with the server.
--- a/scripts/monit/freeradius.monitrc
+++ b/scripts/monit/freeradius.monitrc
@@ -8,9 +8,9 @@
 #  Totalmem limit should be lowered to 200.0 if none of the 
 #  interpreted language modules or rlm_cache are being used.
 #
-check process radiusd with pidfile /var/run/radiusd/radiusd.pid
-   start program = "/etc/init.d/radiusd start"
-   stop program = "/etc/init.d/radiusd stop"
+check process freeradius with pidfile /var/run/freeradius/freeradius.pid
+   start program = "/etc/init.d/freeradius start"
+   stop program = "/etc/init.d/freeradius stop"
    if failed host 127.0.0.1 port 1812 type udp protocol radius secret testing123 then alert
    if failed host 127.0.0.1 port 1813 type udp protocol radius secret testing123 then alert
    if cpu > 95% for 2 cycles then alert
